exercises/rotational-cipher/Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+name = "rotational-cipher"
+version = "0.0.0"
+authors = ["Chad Baxter <cbaxter@mail.umw.edu>"]
```

Comment:
Since there is prior precedent, I don't object, though I won't speak for anyone else who may feel more strongly.

I will note that you will probably want to put yourself as the source when adding the exercise to x-common, as there is a source and source_url in metadata - example at https://github.com/exercism/x-common/blob/master/exercises/paasio/metadata.yml#L3

Unless you think that should be a wikipedia link